### PR TITLE
CT-1446 test eb on byodb

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: CT-1446
KBC:  https://github.com/keboola/connection/pull/5077

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+        $this->initEvents($this->_client);
+        $token = $this->_client->verifyToken();
+
+        // check that this project does not have external buckets feature enabled
+        $this->assertFalse(in_array('external-buckets', $token['owner']['features']));
+        $guide = $this->_client->registerBucketGuide([self::TESTDB, self::TESTSCHEMA], 'snowflake');
+
+        $guideEploded = explode("\n", $guide['markdown']);
+        $host = getenv('SNOWFLAKE_HOST');
```
Comment (jirkasemmler) [inline tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php:40]: chtel jsem udelat `getevn() || throw` ale to prej nejde v php7.4

File: tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Keboola\Test\Backend\ExternalBuckets;
+
+use http\Exception\RuntimeException;
```
Comment (romanbracinik) [inline tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php:5]: toto sa nepouziva

**Review by romanbracinik — APPROVED** [review]
Ok, testy som pustal uz pri review connection

File: tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php (hunk 2 above)
Comment (jirkasemmler) [reply tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php]: super, jeste jsem zrusil ta TODO, protoze https://github.com/keboola/connection/pull/5079 je uz mergnute. lokalne mi to proslo na novem projektu (to je zadouci, novy projekt je potencialne nebezpecny)
image

**Pull request merged**